### PR TITLE
Issue #463

### PR DESCRIPTION
**Pull request by eli9701**
this is a tad screwed because i couldn't figure some of it out

### CODE REVIEW

**Comment by eli9701** [comment]
fixes-#464

File: www/artists/thall/index.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
```
Comment (gemohandy) [inline www/artists/thall/index.html]: Why are you touching thall? Who is thall? thall doesn't have a page on the database. Is this supposed to be a new artist? If so, that should be a different pull request on a different branch. If not, then what is this?!?

File: www/artists/thall/index.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="stylesheet.css">
+
+</head>
+<body>
+  <li class="home">  <a href= "index.html"> Home</a> </li>
```
Comment (gemohandy) [inline www/artists/thall/index.html]: You aren't allowed to have list items outside of either an unordered list or an ordered list. These list items are outside of an ordered or unordered list. Do you see the problem?

File: www/artists/thall/index.html [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  <li class="photographs"> <a href= photograpghs.html>photographs</a> </li>
+  <li class="saskartists"> <a href= https://saskartists.ca/>Sask artists</a> </li>
+  <h1 class="Name">Don hall</h1>
```
Comment (gemohandy) [inline www/artists/thall/index.html]: This claims his name is Don Hall. We don't have records for a Don Hall. More importantly, Don Hall would have a file folder of dhall, not thall.

**Comment by gemohandy** [comment]
@synixebrett - If Will is no longer working on SaskArtists, should this be reassigned?
